### PR TITLE
15969 Update ApprovalType

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#15969

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/approval-type/ApprovalType.vue [hunk 1]
```diff
@@ -237,20 +235,6 @@ export default class ApprovalType extends Vue {
     this.validate()
     return applicationDate
   }
-
-  /* Set court order number to empty if approved by registrar option is selected. */
- @Watch('courtOrderNumberText')
-  private setCourtOrderNumber () {
-    // When going from valid to invalid (boundaries), the form's validity doesn't update immediately.
```
Comment (vysakh-menon-aot) [inline src/components/approval-type/ApprovalType.vue:244]: @JazzarKarim Do I need to test any specific scenario to check this?

**Review by severinbeauvais — COMMENTED** [review]
Did you test locally changing from one approval type to the other? (And checking the values emitted?)

**Comment by vysakh-menon-aot** [comment]
> Did you test locally changing from one approval type to the other? (And checking the values emitted?)

Yes I tested it within create-ui and edit-ui

**Review by severinbeauvais — APPROVED** [review]

File: src/components/approval-type/ApprovalType.vue (hunk 1 above)
Comment (JazzarKarim) [reply src/components/approval-type/ApprovalType.vue:244]: Yes Vysakh. Can you please test the following scenarios:
- When going from a court order length of 4 to 5 and vice versa.
- When going from court order length of 20 to 21 and vice versa.

The reason is that the component sometimes doesn't register the immediately the correct invalid state of the form when going from a valid state. Please let me know if you are able to test these scenarios.

Comment (JazzarKarim) [reply src/components/approval-type/ApprovalType.vue:244]: Marking this conversation as resolved after our call 👍

**Review by JazzarKarim — APPROVED** [review]

**Pull request merged**